develop/tutorials/articles/192-item-selector/03-customizing-item-selector-views.markdown

```diff
@@ -0,0 +1,390 @@
+# Creating Custom Selection Views for the Item Selector [](id=creating-custom-selection-views-for-the-item-selector)
```

Comment:
@robertoDiaz @sergiogonzalez What is the purpose of the Display Context class? Is this needed to create a custom view? For example this https://github.com/liferay/liferay-portal/blob/master/modules/apps/collaboration/item-selector/item-selector-upload-web/src/main/java/com/liferay/item/selector/upload/web/internal/display/context/ItemSelectorUploadViewDisplayContext.java class is imported in its https://github.com/liferay/liferay-portal/blob/master/modules/apps/collaboration/item-selector/item-selector-upload-web/src/main/java/com/liferay/item/selector/upload/web/internal/ItemSelectorUploadView.java class.